src/vaadin-message-input.js

```diff
@@ -21,6 +24,17 @@ import '@vaadin/vaadin-text-field/src/vaadin-text-area.js';
  * @mixes ElementMixin
  */
 class MessageInputElement extends ElementMixin(ThemableMixin(PolymerElement)) {
+  static get properties() {
+    return {
+      /**
+       * Current content of the text input field
+       */
+      text: {
```

Comment:
I would call this `value`, which is the convention for input elements. `text` usually refers to static text content, like labels and such.

Reply:
Renamed to value.